content/en/docs/concepts/traffic-management/index.md

```diff
@@ -885,12 +511,10 @@ spec:
 {{< /text >}}
 
 This gateway configuration lets HTTPS traffic from `ext-host` into the mesh on
```

Comment:
`ext-host` is a silly example of an external host name. Maybe change to something like `ext-host.example.com`?

Reply:
Good spot. Fixed.